packages/datatrak-web/src/views/OfflinePage.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button as UIButton } from '@tupaia/ui-components';
+import { Link as RouterLink } from 'react-router-dom';
+import { Typography } from '@material-ui/core';
+import { HEADER_HEIGHT } from '../constants';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+`;
+
+const HeaderContainer = styled.div`
```

Comment:
Is this semantically header?

```suggestion
const HeaderContainer = styled.header`
```